docs/spec_v6.md

```diff
@@ -0,0 +1,259 @@
+---
+id: "spec_v6"
+title: "Twirp Wire Protocol"
+sidebar_label: "Version 6 (Proposed Draft)"
+---
+
+This document is a draft of a new version ("v6") of the Twirp wire
+protocol over HTTP. This draft is EXPERIMENTAL and subject to change.
+v6 of Twirp is not released or considered stable; instead, this
+document lists planned changes which may be available in prerelease
+distributions.
+
+## Changes from v5
+
+### URL scheme
+
+In [v5](./PROTOCOL.md), URLs followed this format:
+
+**URL ::= Base-URL "/twirp/" [ Package "." ] Service "/" Method**
+
+In v6, the "/twirp/" prefix is removed:
+
+**URL ::= Base-URL "/" [ Package "." ] Service "/" Method**
+
+The "/twirp/" prefix is removed for three reasons:
+
+ - Trademark concerns: some very large organizations don't want to
+   take any legal risks and are concerned that "twirp" could become
+   trademarked.
+ - Feels like advertising: To some users, putting "twirp" in all your
+   routes feels like it's just supposed to pump Twirp's brand, and
+   provides no value back to users.
+ - Homophonous with "twerp": In some Very Serious settings (like
+   government websites), it's not okay that "twirp" sounds like
+   "twerp", which means something like "insignificant pest."
```

Comment:
This is a concern only for whoever is updating from v5 to v6, I think it is better to keep the v6 (latest) spec clear without this. It may be enough with adding this info in the Release Notes for v6, or if you want to keep this info in the spec is better to add it in the v5 spec (as a notice that it changed after v6).

Reply:
Yeah, I'll move this sort of thing into a changelog doc that will go unpublished until we release v6.